robot-custom-integration/sl_python_robot/SLListener.py

```diff
@@ -42,19 +45,22 @@ def start_suite(self, suite, result):
                 test.body.create_keyword(name="SKIP")
 
         tests_for_execution = list(all_tests - self.excluded_tests)
-        print(f'>>>> Tests for execution: {tests_for_execution}')
+        print(f'{SEALIGHTS_LOG_TAG} {len(tests_for_execution)} Tests for execution: {tests_for_execution}')
 
     def end_suite(self, date, result):
         if not self.test_session_id:
             return
         """Collect and report test results to SeaLights including start and end time"""
         test_results = self.build_test_results(result)
         if test_results:
-            requests.post(self.get_session_url(), json=test_results, headers=self.get_header())
-            print(f'>>>> {SEALIGHTS_LOG_TAG} Results: {test_results}')
+            print(f'{SEALIGHTS_LOG_TAG} {len(test_results)} Results to send: {test_results}')
```

Comment:
word "Results" should not be capitalized as it's not the first one in the sentence anymore.